src/dialects/mariadb/query-generator.js

```diff
@@ -1,14 +1,57 @@
 'use strict';
 
+import { normalizeDataType } from '../abstract/data-types-utils';
+
 const { MySqlQueryGenerator } = require('../mysql/query-generator');
 const _ = require('lodash');
+const Utils = require('../../utils');
 
 export class MariaDbQueryGenerator extends MySqlQueryGenerator {
 
   _getTechnicalSchemaNames() {
     return ['MYSQL', 'INFORMATION_SCHEMA', 'PERFORMANCE_SCHEMA', 'mysql', 'information_schema', 'performance_schema'];
   }
 
+  addColumnQuery(table, key, dataType, options) {
+    options = options || {};
+
+    const ifNotExists = options.ifNotExists ? ' IF NOT EXISTS' : '';
+
+    dataType = {
+      ...dataType,
+      type: normalizeDataType(dataType.type, this.dialect),
+    };
+
+    return Utils.joinSQLFragments([
```

Comment:
Can we use `joinSQLFragments` directly?
By using `import { joinSQLFragments } from '../../utils/join-sql-fragments';`

No need to refactor existing usages, just the new ones is fine

Reply:
Duplicate